job-dsls/jobs/pr_jobs.groovy

```diff
@@ -8,8 +8,8 @@ def final DEFAULTS = [
         branch                 : "master",
         timeoutMins            : 60,
         label                  : "rhel7 && mem8g",
-        upstreamMvnArgs        : "-B -e -T1C -DskipTests -Dgwt.compiler.skip=true -Denforcer.skip=true -Dcheckstyle.skip=true -Dfindbugs.skip=true -Drevapi.skip=true clean install",
-        mvnGoals               : "-e -nsu -fae -B -T1C -Pwildfly11 clean install",
+        upstreamMvnArgs        : "-B -e -T1C -DskipTests -Dgwt.compiler.skip=true -Denforcer.skip=true -Dcheckstyle.skip=true -Dfindbugs.skip=true -Drevapi.skip=true -Dmaven.javadoc.skip=true clean install",
+        mvnGoals               : "-B -e -T1c -nsu -fae -Pwildfly11 clean install",
```

Comment:
Not adding the javadoc skip here, as we might want to build javadoc somewhere (?) Can anything go wrong if we don't build it anywhere (some kind of fata javadoc failure)? Not sure..

Reply:
That's a good question. We have daily builds, so if we decide to disable it for PRs, doc issues should caught later. On the other side for single module build javadoc doesn't have so big footprint. I don't know what is better but would vote for leaving javadoc for PRs.

Reply:
Ok, I'll leave it as it currently is = disable everywhere in full downstream builds, enabled in single repo build.